dags/atd_visionzero_cr3_process_missing_pdfs.py

```diff
@@ -0,0 +1,84 @@
+import os
+
+from airflow.models import DAG
+from airflow.operators.docker_operator import DockerOperator
+from pendulum import datetime, duration
+
+from utils.onepassword import get_env_vars_task
+from utils.knack import get_date_filter_arg
+from utils.slack_operator import task_fail_slack_alert
+
+DEPLOYMENT_ENVIRONMENT = os.getenv("ENVIRONMENT", "development")
+
+DEFAULT_ARGS = {
+    "owner": "airflow",
+    "depends_on_past": False,
+    "start_date": datetime(2015, 1, 1, tz="America/Chicago"),
+    "email_on_failure": False,
+    "email_on_retry": False,
+    "retries": 0,
+    "retry_delay": duration(minutes=5),
+    "on_failure_callback": task_fail_slack_alert,
+}
+
+# secrets atd_visionzero_hasura_sql_production
+
+REQUIRED_SECRETS = {
+    "HASURA_ENDPOINT": {
+        "opitem": "Vision Zero graphql-engine Endpoints",
+        "opfield": f"{DEPLOYMENT_ENVIRONMENT}.Query Endpoint",
+    },
+    "HASURA_ADMIN_KEY": {
+        "opitem": "Vision Zero graphql-engine Endpoints",
+        "opfield": f"{DEPLOYMENT_ENVIRONMENT}.Admin Key",
+    },
+    "AWS_BUCKET_NAME": {
+        "opitem": "CRIS CR3 Download",
+        "opfield": f"{DEPLOYMENT_ENVIRONMENT}.AWS_CRIS_CR3_BUCKET_NAME",
+    },
+    "AWS_BUCKET_ENVIRONMENT": {
+        "opitem": "CRIS CR3 Download",
+        "opfield": f"{DEPLOYMENT_ENVIRONMENT}.AWS_CRIS_CR3_BUCKET_PATH_ENVIRONMENT",
+    },
+}
+
+with DAG(
+    dag_id="vision_zero_process_missing_pdfs",
+    description="Execute housekeeping routine manage missing or malformed CR3 PDFs",
+    default_args=DEFAULT_ARGS,
+    schedule_interval="*/30 * * * *"
+    if DEPLOYMENT_ENVIRONMENT == "production"
+    else None,
+    dagrun_timeout=duration(minutes=15),
+    tags=["repo:atd_vz_data", "vision-zero", "cr3", "pdf"],
+    catchup=False,
+) as dag:
+    docker_image = "atddocker/vz-pdf-maintenance:latest"
```

Comment:
Should this be tagged "production" instead of latest?
Also, I don't see this docker image name in the vz-data repo, but I do see one named `atddocker/process_missing_cr3_pdfs`

Reply:
re `:production`, it absolutely should. Thank you for catching that! 

Thank you also for catching that I had image name mismatch between this repo and the VZ one. I've fixed that.